test/plugins/moveElemsAttrsToGroup.08.svg.txt

```diff
@@ -0,0 +1,47 @@
+Don't move transform if there is a filter attribute on group.
```

Comment:
Can we keep tests to more of an MRE (Minimal, Reproducible Example).

For example, this should suffice, which is just a minimal version of the SVG:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
  <defs>
    <filter id="a" x="17" y="13" width="12" height="10" filterUnits="userSpaceOnUse">
      <feGaussianBlur stdDeviation=".01"/>
    </filter>
  </defs>
  <g filter="url(#a)">
    <rect x="19" y="12" width="14" height="6" transform="rotate(31 19 12.79)"/>
    <rect x="19" y="12" width="14" height="6" transform="rotate(31 19 12.79)"/>
  </g>
</svg>
```

Reply:
I updated with your test case, but with the addition of rx="3" to rects, since that seems to be necessary to see a difference with the unfixed version.